src/components/BorrowerProfile/LendCta.vue

```diff
@@ -1,107 +1,140 @@
 <template>
-	<div>
-		<div class="tw-fixed
-					tw-left-0
-					md:tw-relative
-					md:tw-left-unset
-					tw-bottom-0
-					tw-right-0
-					md:tw-rounded-b
-					md:tw-rounded-t-none
-					lg:tw-rounded-t
-					tw-bg-white
-					tw-border
-					tw-bt-1
-					tw-border-gray-300
-					md:tw-border-none
-					tw-pt-1
-					tw-px-2.5
-					md:tw-pt-0
-					md:tw-pb-2
-					lg:tw-px-4
-					lg:tw-py-1"
+	<div ref="wrapper"
+		:class="['lg:tw-mb-1.5', { 'md:tw-px-4': isSticky }]"
+		:style="wrapperStyle"
+	>
+		<div
+			:class="[
+				'tw-w-full tw-z-10',
+				'tw-flex tw-flex-col',
+				'tw-fixed tw-left-0 tw-bottom-0',
+				{
+					'md:tw-static': !isSticky,
+				},
+				'lg:tw-static',
+			]"
 		>
-			<p class="tw-text-h3 tw-pt-3 lg:tw-mb-3 tw-hidden lg:tw-inline-block">
-				{{ lgScreenheadline }}
-			</p>
-			<span class="tw-flex tw-pb-1 lg:tw-pb-3">
-				<kv-ui-select
-					v-if="hideShowLendDropdown"
-					id="LoanAmountDropdown"
-					class="tw-pr-2.5 tw--mb-2"
-					v-model="selectedOption"
+			<kv-grid
+				:class="[
+					'tw-grid-cols-12 tw-gap-x-2',
+					'tw-px-2.5',
+					'tw-bg-white',
+					'tw-border-t tw-border-gray-300',
+					{
+						'md:tw-rounded-b md:tw-border-none': !isSticky,
+						'md:tw-px-3': !isSticky,
+						'md:tw-px-4': isSticky,
+					},
+					'lg:tw-rounded-t',
+					'lg:tw-px-4',
+				]"
+			>
+				<div
+					:class="[
+						'tw-pt-1',
+						'tw-col-span-12',
+						{
+							'md:tw-pt-0 md:tw-pb-2': !isSticky,
+							'md:tw-col-start-2 md:tw-col-span-10': isSticky,
+						},
+						'lg:tw-col-span-12',
+						'lg:tw-py-1',
+					]"
 				>
-					<option
-						v-for="priceOption in prices"
-						:key="priceOption"
-						:value="priceOption"
+					<p class="tw-text-h3 tw-pt-3 lg:tw-mb-3 tw-hidden lg:tw-inline-block">
+						{{ lgScreenheadline }}
+					</p>
+					<span class="tw-flex tw-pb-1 lg:tw-pb-3">
+						<kv-ui-select
```

Comment:
This select will need a label element for a11y, maybe something like `<label for="LoanAmountDropdown" class="tw-sr-only">Lend amount</label>`